src/item/component/RenderOffsetsComponent.php

```diff
@@ -3,40 +3,46 @@
 
 namespace customiesdevs\customies\item\component;
 
-final class RenderOffsetsComponent implements ItemComponent {
 
-	private int $textureWidth;
-	private int $textureHeight;
-	private bool $handEquipped;
+final class RenderOffsetsComponent implements ItemComponent { //thanks to https://github.com/MedicalJewel105/bedrock-render-offsets-generator/blob/main/main.py
 
-	public function __construct(int $textureWidth, int $textureHeight, bool $handEquipped = false) {
-		$this->textureWidth = $textureWidth;
-		$this->textureHeight = $textureHeight;
-		$this->handEquipped = $handEquipped;
+	private int $textureSize;
+
+	public function __construct(int $textureSize) {
+		$this->textureSize = $textureSize;
 	}
 
 	public function getName(): string {
 		return "minecraft:render_offsets";
 	}
 
 	public function getValue(): array {
-		$horizontal = ($this->handEquipped ? 0.075 : 0.1) / ($this->textureWidth / 16);
-		$vertical = ($this->handEquipped ? 0.125 : 0.1) / ($this->textureHeight / 16);
```

Comment:
Should handEquipped be removed? Does it no longer do anything different?

Reply:
I've removed it yes, based on the source (in comment) it doesnt change anything. I tried in game and have this since 4/5 months, nobody seems to notice anything wrong.. I guess it's useless now ?